src/utils/currencies.ts

```diff
@@ -41,7 +41,7 @@ function isValidCurrencySymbolKey(key: string): key is CurrencySymbolKey {
 /**
  * Gets the string representation of a token amount with fixed decimals
  */
-function getFixedTokenAmount(amount: string, decimals = 18): string {
+function getFixedTokenAmount(amount: string | undefined, decimals = 18): string {
```

Comment:
Because I didn't remove this TODO https://github.com/hermeznetwork/wallet-ui/blob/0c79527637534cfe5b1532b0fa3864b0a2189c16/src/utils/transactions.ts#L22, I do it now and remove this change